app/scripts/views/settings.js

```diff
@@ -150,9 +150,7 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
     },
 
     beforeDestroy: function () {
-      $('body.settings').fadeOut(this.FADE_OUT_SETTINGS_MS, function (){
-        $('body').removeClass('settings').show();
-      });
+      $('body').removeClass('settings');
```

Comment:
Without this there is an issue rendering away from the view. You can see the "flicker", when the  "settings" opacity goes away. 
![set-pass](https://cloud.githubusercontent.com/assets/128755/9529000/c6f6ea9a-4cc5-11e5-99b1-ad966aab56e5.gif)

@zaach any ideas what to do about this? how important is the fade out?